code/modules/reagents/chemistry_reagents/other.dm

```diff
@@ -792,3 +792,40 @@
 		if(volume < 3) return
 		if(!(locate(/obj/effect/decal/cleanable/blackgoo) in T))
 			new /obj/effect/decal/cleanable/blackgoo(T)
+
+
+/datum/reagent/xeno_neurotoxin
+	name = "Neurotoxin"
+	id = "xeno_toxin"
+	description = "A debilitating nerve toxin. Impedes motor control. Causes temporary blindness, hallucinations and deafness at higher doses."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	custom_metabolism = 0.5 // Fast meta rate.
+	overdose = REAGENTS_OVERDOSE
+	overdose_critical = REAGENTS_OVERDOSE_CRITICAL
+
+/datum/reagent/xeno_neurotoxin/on_mob_life(mob/living/M)
+	. = ..()
+	if(!.)
+		return
+	var/halloss_damage = volume * 2 * REM
+	M.apply_damage(halloss_damage, HALLOSS) //1st level neurotoxin effects: halloss/pain
+	switch(volume)
+		if(5 to INFINITY) //2nd level neurotoxin effects: screen shake, drug overlay, stuttering, minor toxin damage
+			M.druggy += 1.5
+			M.stuttering += 1.5
+		if(15 to INFINITY) //3rd level neurotoxin effects: blindness, deafness
+			M.ear_deaf += 1.5
+			M.eye_blurry += 1.5
+			M.eye_blind += 1.5
+		if(20 to INFINITY) //4th level neurotoxin effects: paralysis and toxin damage
+			M.stunned += 1
+			M.KnockDown(1)
+
+/datum/reagent/xeno_neurotoxin/on_overdose(mob/living/M)
+		var/toxin_damage = volume * 0.1 * REM
+		M.adjustToxLoss(1 + toxin_damage * REM) //Overdose starts applying more toxin damage
+
+/datum/reagent/xeno_neurotoxin/on_overdose_critical(mob/living/M)
+		var/toxin_damage = volume * 0.1 * REM
+		M.adjustToxLoss(2 + toxin_damage * REM) //Even more toxin damage
```

Comment:
REM = REAGENTS_METABOLISM / 0.4 or 0.2 / 0.4 = 0.5

So 2 + volume * 0.5 * 0.1.

I think I'll just make it completely dependent on volume rather than featuring this flat amount.